__tests__/app.test.js

```diff
@@ -38,3 +39,41 @@ describe.only("3. GET /api/topics", () => {
       });
   });
 });
+
+describe("4. GET /api/articles", () => {
+  it("status:200, should responds with an array of articles with the creation date in descending order", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body }) => {
+        const { articles } = body;
+        expect(articles).toBeInstanceOf(Array);
```

Comment:
You are using a .forEach() array method in this test, so is checking if articles is an array really necessary?

Reply:
Because I'm afraid if in any case articles is not an array, it will failed on line 50 instead of while carry out .forEach()